framework/includes/option-types/wp-editor/class-fw-option-type-wp-editor.php

```diff
@@ -75,8 +75,8 @@ public function _filter_disable_default_init($mceInit, $editor_id){
 	 * @internal
 	 */
 	protected function _render( $id, $option, $data ) {
-		if ($option['shortcodes_list'] === 'default') {
-			$option['shortcodes_list'] = $this->get_default_shortcodes_list();
+		if ($option['shortcodes']) {
```

Comment:
`$option['shortcodes']` can be `array` which also evaluates to true, so the user will not be able to set custom shortcodes list because this `if` will be always executed. Should be `if ($option['shortcodes'] === true) {`